Pathogens/Arbo/ETL/arbo_etl_main.py

```diff
@@ -133,20 +133,14 @@ def main():
     # print(records_df.dtypes)
 
     print("[STEP] generating lat and lng values")
-    # calculate the lat and lng values for each of the rows.
-    # TODO: Not working correctly yet
+
     records_df[['latitude', 'longitude']] = records_df.apply(lambda row:
-                                                             pd.Series(get_lng_lat(
-                                                                 ','.join(filter(None, [row['city'], row['state']])),
-                                                                 'place'), dtype='object')
-                                                             if pd.notnull(row['city']) and pd.notnull(row['state'])
-                                                             else (pd.Series(
-                                                                 get_lng_lat(row['state'], 'region'),
-                                                                 dtype='object') if pd.notnull(
-                                                                 row['state'])
-                                                                   else pd.Series(
-                                                                 get_lng_lat(row['country'], 'country'),
-                                                                 dtype='object')), axis=1)
+        pd.Series(get_city_lat_lng(
```

Comment:
I think it would be valuable to introduce a jitter pins functionality here this time around itself instead of waiting for the future. 

Would you be able to do that? essentially add or subtract up to 5 degrees from the Latitude and longitude of each estimate so they are all a little more scattered. 

Can reference the jitter_pins function if you need it!

Reply:
Yeah this is a sick idea. Definitely necessary, I didn't realize how many cities had a whole bunch of studies :+1:

I tried some values for what the maximum jitter should be and I really really liked the way the map looked on 0.1. When I turned the jitter up too high I didn't like what it did to the map but I definitely get that these things are subjective so if you want more jitter let's turn it up :chart_with_upwards_trend: but if you also like 0.1 let's send it :mailbox: